Maven throws NPE on bloopInstall

I've been looking at Bloop and Metals to try out at work. So far, I've had no luck due to #862. Now Bloop 1.3.0 came out I decided to give it another try. Unfortunately, I'm still getting an NPE, but a different one:

```
└ ❯ mvn ch.epfl.scala:maven-bloop_2.10:1.3.0:bloopInstall -e
[INFO] Error stacktraces are turned on.
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------< mycompany:my-work-project >-------------------
[INFO] Building my-work-project LOCAL
[INFO] --------------------------------[ jar ]---------------------------------
[INFO]
[INFO] --- maven-bloop_2.10:1.3.0:bloopInstall (default-cli) @ my-work-project ---
[INFO] Using empty launcher: no run setup for my-work-project.
[INFO] compiler plugin: BasicArtifact(org.wartremover,wartremover_2.12,2.2.1,null)
[INFO] Generated .bloop/my-work-project.json
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.656 s
[INFO] Finished at: 2019-06-07T12:36:38+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal ch.epfl.scala:maven-bloop_2.10:1.3.0:bloopInstall (default-cli) on project my-work-project: Execution default-cli of goal ch.epfl.scala:maven-bloop_2.10:1.3.0:bloopInstall failed.: NullPointerException -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal ch.epfl.scala:maven-bloop_2.10:1.3.0:bloopInstall (default-cli) on project my-work-project: Execution default-cli of goal ch.epfl.scala:maven-bloop_2.10:1.3.0:bloopInstall failed.
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.apache.maven.plugin.PluginExecutionException: Execution default-cli of goal ch.epfl.scala:maven-bloop_2.10:1.3.0:bloopInstall failed.
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:148)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.lang.NullPointerException
    at io.circe.Printer$PrintingFolder.onString (Printer.scala:272)
    at io.circe.Printer$PrintingFolder.onString (Printer.scala:256)
    at io.circe.Json$JString.foldWith (Json.scala:299)
    at io.circe.Printer$PrintingFolder.onArray (Printer.scala:318)
    at io.circe.Printer$PrintingFolder.onArray (Printer.scala:256)
    at io.circe.Json$JArray.foldWith (Json.scala:330)
    at io.circe.JsonObject$LinkedHashMapJsonObject.appendToFolder (JsonObject.scala:352)
    at io.circe.Printer$PrintingFolder.onObject (Printer.scala:326)
    at io.circe.Printer$PrintingFolder.onObject (Printer.scala:256)
    at io.circe.Json$JObject.foldWith (Json.scala:361)
    at io.circe.JsonObject$LinkedHashMapJsonObject.appendToFolder (JsonObject.scala:352)
    at io.circe.Printer$PrintingFolder.onObject (Printer.scala:326)
    at io.circe.Printer$PrintingFolder.onObject (Printer.scala:256)
    at io.circe.Json$JObject.foldWith (Json.scala:361)
    at io.circe.JsonObject$LinkedHashMapJsonObject.appendToFolder (JsonObject.scala:352)
    at io.circe.Printer$PrintingFolder.onObject (Printer.scala:326)
    at io.circe.Printer$PrintingFolder.onObject (Printer.scala:256)
    at io.circe.Json$JObject.foldWith (Json.scala:361)
    at io.circe.Printer.pretty (Printer.scala:190)
    at bloop.config.package$.toStr (package.scala:12)
    at bloop.config.package$.write (package.scala:16)
    at bloop.integrations.maven.MojoImplementation$.writeConfig$1 (MojoImplementation.scala:142)
    at bloop.integrations.maven.MojoImplementation$.writeCompileAndTestConfiguration (MojoImplementation.scala:145)
    at bloop.integrations.maven.MojoImplementation.writeCompileAndTestConfiguration (MojoImplementation.scala)
    at bloop.integrations.maven.BloopMojo.execute (BloopMojo.java:72)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:956)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:288)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:192)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[ERROR]
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

We use `net.alchim31.maven:scala-maven-plugin:3.2.2`. There are some Java files in the project, but the projects are mostly Scala.

--- a/integrations/maven-bloop/src/main/scala/bloop/integrations/maven/MojoImplementation.scala
+++ b/integrations/maven-bloop/src/main/scala/bloop/integrations/maven/MojoImplementation.scala
@@ -73,7 +73,7 @@ object MojoImplementation {
 
     val compileSetup = mojo.getCompileSetup()
     val allScalaJars = mojo.getAllScalaJars().map(abs).toList
-    val scalacArgs = mojo.getScalacArgs().asScala.toList
+    val scalacArgs = mojo.getScalacArgs().asScala.toList.filter(_ != null)
 
     def writeConfig(
         sourceDirs0: Seq[File],

